compiler/noirc_evaluator/src/ssa/acir_gen/mod.rs

```diff
@@ -1103,15 +1100,14 @@ impl<'a> Context<'a> {
         index: ValueId,
         store_value: Option<ValueId>,
     ) -> Result<bool, RuntimeError> {
-        let array_id = dfg.resolve(array);
-        let array_typ = dfg.type_of_value(array_id);
```

Comment:
Same question here about not resolving things.

Reply:
I'll add it back, I'm not sure why I thought `type_of_value` doesn't need it.